src/initialization.js

```diff
@@ -69,21 +69,31 @@ var initialization = {
 
         common.consentPayloadDefaults =
             common.consentHandler.getConsentSettings();
-        var initialConsentState = common.consentHandler.getUserConsentState();
 
-        var defaultConsentPayload =
-            common.consentHandler.generateConsentStatePayloadFromMappings(
-                initialConsentState,
-                common.consentMappings
-            );
+        var defaultConsentPayload = common.cloneObject(common.consentPayloadDefaults)
+        var updatedConsentState = common.consentHandler.getUserConsentState();
+        var updatedDefaultConsentPayload =
+        common.consentHandler.generateConsentStatePayloadFromMappings(
+            updatedConsentState,
+            common.consentMappings
+        );
 
         if (!common.isEmpty(defaultConsentPayload)) {
             common.consentPayloadAsString = JSON.stringify(
                 defaultConsentPayload
             );
 
             common.sendConsent('default', defaultConsentPayload);
+        } else if (!common.isEmpty(updatedDefaultConsentPayload)) {
+            common.consentPayloadAsString = JSON.stringify(
+                updatedDefaultConsentPayload
+            );
+
+            common.sendConsent('default', updatedDefaultConsentPayload);
```

Comment:
I can probably make a function in common for lines 82-86 and 88-92 and call it here instead of having the same repeated function since they're doing the same actions just with two different variables